banzai/tests/test_end_to_end.py

```diff
@@ -28,48 +26,61 @@
 
 app.conf.update(CELERY_TASK_ALWAYS_EAGER=True)
 
+TEST_PACKAGE = 'banzai.tests'
+TEST_FRAMES = ascii.read(pkg_resources.resource_filename(TEST_PACKAGE, 'data/test_data.dat'))
+
+PRECAL_FRAMES = ascii.read(pkg_resources.resource_filename(TEST_PACKAGE, 'data/test_precals.dat'))
+
 DATA_ROOT = os.path.join(os.sep, 'archive', 'engineering')
-SITES = [os.path.basename(site_path) for site_path in glob(os.path.join(DATA_ROOT, '???'))]
-INSTRUMENTS = [os.path.join(site, os.path.basename(instrument_path)) for site in SITES
-               for instrument_path in glob(os.path.join(os.path.join(DATA_ROOT, site, '*')))]
+# Use the LCO filenaming convention to infer the sites
+SITES = set([frame[:3] for frame in TEST_FRAMES['filename']])
+INSTRUMENTS = set([os.path.join(frame[:3], frame.split('-')[1]) for frame in TEST_FRAMES['filename']])
```

Comment:
Now that the .dat file has the site/instrument names in it, can't we use those columns instead of the filename?